Pipfile

```diff
@@ -4,7 +4,7 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
-linkml = ">=1.0.4"
+linkml = ">=1.0.3"
```

Comment:
I forget what was added in linkml version `1.0.4`. But the higher version might be needed.